lib/find-view.js

```diff
@@ -23,6 +23,11 @@ class FindView {
     this.updateOptionViews();
     this.updateReplaceEnablement();
     this.createWrapIcon();
+
+    if (atom.textEditors) {
+      var disposable = atom.textEditors.add(this.findEditor);
+      this.findEditor.onDidDestroy(() => disposable.dispose());
```

Comment:
I don't think we destroy these editors when the package deactivates, so I'm not sure this will work as intended. Since these editors live for the entire time the package is active and we never create more than one find panel, it might be better to just dispose the registration when this package deactivates.